### PR TITLE
Add Legacy points from X2PO

### PR DESCRIPTION
**Pull request by SogeMoge**

### CODE REVIEW

File: X2PO/mar2024/galacticrepublic.json [hunk 1]
```diff
@@ -0,0 +1,1727 @@
+{
+    "ARC-170 Starfighter": {
+        "sinker": {
+            "name": "\u201cSinker\u201d",
```
Comment (eirikmun) [inline X2PO/mar2024/galacticrepublic.json]: I know the script I sent you created these (by converting to utf8). But it's better to have them in ascii format directly.
Could you alter the script to create that?

File: X2PO/last/firstorder.json
Comment (eirikmun) [inline X2PO/last/firstorder.json:1]: The last folder really isn't needed (it's not referenced in your points-revisions.json file.

**Review by eirikmun — CHANGES_REQUESTED** [review]

File: X2PO/last/firstorder.json
Comment (SogeMoge) [reply X2PO/last/firstorder.json:1]: Legacy will have a regular points update soon, this week, it's a placeholder

File: X2PO/mar2024/galacticrepublic.json (hunk 1 above)
Comment (SogeMoge) [reply X2PO/mar2024/galacticrepublic.json]: I can do it not sooner than Wednesday. Maybe sooner, can't commit fully. I see that AMG and XWA points also have UTF-8, should be ok for now?

File: X2PO/last/firstorder.json
Comment (SogeMoge) [reply X2PO/last/firstorder.json:1]: it should be here 
```
{
    "current_revision": "X2PO/mar2024/revision.json",
    "revision_history": [
        "X2PO/last/revision.json",
        "X2PO/mar2024/revision.json"
    ]
}
```

File: X2PO/mar2024/galacticrepublic.json (hunk 1 above)
Comment (SogeMoge) [reply X2PO/mar2024/galacticrepublic.json]: Done

**Review by SogeMoge — COMMENTED** [review]
Special characters are preserved as is

File: X2PO/last/revision.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+    "effective_date": "2024-03-26",
```
Comment (eirikmun) [inline X2PO/last/revision.json:2]: Is this correct?

**Review by eirikmun — APPROVED** [review]

File: X2PO/last/revision.json (hunk 2 above)
Comment (SogeMoge) [reply X2PO/last/revision.json:2]: aha, removed "last" in second PR

**Pull request merged**